distribution/openrct2.d.ts

```diff
@@ -2076,6 +2076,61 @@ declare global {
          * The min chain lift speed for this ride in miles per hour.
          */
         readonly minLiftHillSpeed: number;
+
+        /**
+         * The max speed in miles per hour.
+         */
+        readonly maxSpeed: number;
+
+        /**
+         * The average speed in miles per hour.
+         */
+        readonly averageSpeed: number;
+
+        /**
+         * The ride time in seconds.
+         */
+        readonly rideTime: number;
+
+        /**
+         * Total length of the ride in meters.
+         */
+        readonly rideLength: number;
+
+        /**
+         * The max positive vertical Gs in hundredths of a G.
+         */
+        readonly maxPositiveVerticalGs: number;
```

Comment:
Hundredths of G is a weird unit. Since we now have floats for the speed stats, why not also have them for the force stats?

Reply:
I figured that since it's ok for excitement/nausea/intensity to get returned as hundredths then it'd be fine if Gs were returned as hundredths as well, but for my uses it'd be more convenient to return as a float anyways. Will change.